test/runtests.jl

```diff
@@ -279,14 +279,19 @@ end
 end
 
 # Test downstream dependencies
-using Pkg
-Pkg.add("Taro")
-
-using Taro
-chmod(joinpath(dirname(dirname(pathof(Taro))),"test","df-test.xlsx"),0o600)
-
-Pkg.test("Taro")
-#include(joinpath(dirname(dirname(pathof(Taro))),"test","runtests.jl"))
+try
```

Comment:
Can't we add it as a test depedency?

Reply:
I added Taro as a test dependency by putting it in extras.

However, Taro is failing to work on Mac and this does not appear to be caused by this PR.